images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -21,13 +21,6 @@ Restore-UserOwner
 
 $markdown = ""
```

Comment:
Does it make sense to get rid of this line in both `SoftwareReport.Generator.ps1` files?